sidebars.js

```diff
@@ -1,91 +1,97 @@
 module.exports = {
-  docs: {
-    'Core concepts': [
-      'overview',
-      {
-        'type': 'category',
-        'label': 'Use cases',
-        'items': [
-          'use-cases-orchestration',
-          'use-cases-distributed-transactions',
-          'use-cases-provisioning',
-          'use-cases-monitoring',
-          'use-cases-pipelines',
-          'use-cases-long-running',
-          'use-cases-dsl',
-          'use-cases-actors',
-        ],
-      },
-      {
-        'type': 'category',
-        'label': 'Learn',
-        'items': [
-          'learn-glossary',
-          'learn-workflows',
-          'learn-activities',
-          'learn-events',
-          'learn-queries',
-          'learn-topology',
-          'learn-task-queues',
-          'learn-server-configuration',
-          'learn-archival',
-          'learn-cross-dc-replication',
-          'learn-workflow-filtering',
-          'learn-cli',
-        ],
-      },
-    ],
-    'Developer guides': [
-      {
-        'type': 'category',
-        'label': 'Getting started',
-        'items': [
-          'installing-server',
-        ],
-      },
-      {
-        'type': 'category',
-        'label': 'Go SDK',
-        'items': [
-          'go-quick-start',
-          'go-sdk-video-tutorial',
-          'go-workers',
-          'go-create-workflows',
-          'go-activities',
-          'go-execute-activity',
-          'go-child-workflows',
-          'go-retries',
-          'go-error-handling',
-          'go-signals',
-          'go-continue-as-new',
-          'go-side-effect',
-          'go-queries',
-          'go-activity-async-completion',
-          'go-workflow-testing',
-          'go-versioning',
-          'go-sessions',
-          'go-distributed-cron',
-          'go-tracing',
-        ],
-      },
-      {
-        'type': 'category',
-        'label': 'Java SDK',
-        'items': [
-          'java-quick-start',
-          'java-workflow-interface',
-          'java-implementing-workflows',
-          'java-starting-workflow-executions',
-          'java-activity-interface',
-          'java-implementing-activities',
-          'java-versioning',
-          'java-distributed-cron',
-        ],
-      },
-    ],
-    'About': [
-      'docs-sla',
-      'license',
-    ],
-  },
+  docs: [
```

Comment:
Here is what this would look like:

![Screen Shot 2020-07-16 at 11 37 38 AM](https://user-images.githubusercontent.com/34380806/87705167-bbcd4f00-c75a-11ea-900b-e1609d97fdfd.png)